client/templates/Report/Formatted/_client_benefits_check.html.twig

```diff
@@ -0,0 +1,64 @@
+{% set translationDomain = "report-client-benefits-check" %}
+{% trans_default_domain translationDomain %}
+
+{% set transOptions = {
+    '%client%': report.client.firstname | e,
+} %}
+
+<div class="section break-before" id="client-benefits-check-section">
+    <h2 class="section-heading">{{ 'common.pageTitle' | trans(transOptions) }}</h2>
+
```

Comment:
Not that it matters much but are we able to sort out the whitespacing here?

Reply:
I've dropped the line break between `box` and `div` but it looks like the markup style for other templates is to leave a break between `h` tags and content. If you're having major eye twitches we could look at changing that up for all templates?

Reply:
Yeah I noticed the `h` tags line break from another file I was looking at I did originally had a comment about that but then removed it :)